base-theme/assets/css/footer.scss

```diff
@@ -0,0 +1,130 @@
+#footer_container {
+  background-color: $footer-background-color;
+  display: flex;
+  padding-left: 120px;
+  padding-top: 33px;
+  padding-right: 40px;
+  justify-content: space-between;
+
+  .abc {
```

Comment:
I wonder if this is a correct name for any class and is it being used somewhere?